src/wallet/asyncrpcoperation_shieldcoinbase.cpp

```diff
@@ -283,7 +287,13 @@ bool ShieldToAddress::operator()(const libzcash::UnifiedAddress &uaddr) const {
     // TODO check if an Orchard address is present, send to it if so.
```

Comment:
Consider improving the error reporting for this case, even though we're not implementing it yet.